pkg/crc/preflight/preflight_checks_nonlinux.go

```diff
@@ -0,0 +1,72 @@
+//go:build !linux
+// +build !linux
+
+package preflight
+
+import (
+	"runtime"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/crc/daemonclient"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/shirou/gopsutil/v3/process"
+)
+
+func killDaemonProcess() error {
+	crcProcessName := "crc"
+	if runtime.GOOS == "windows" {
+		crcProcessName = "crc.exe"
+	}
+	processes, err := process.Processes()
+	if err != nil {
+		return err
+	}
+	for _, process := range processes {
+		// Ignore the error from process.Name() because errors
+		// come from processes which are not owned by the current user
+		// on Mac - https://github.com/shirou/gopsutil/issues/1288
+		name, _ := process.Name()
```

Comment:
Missed that, really should be 
```
name, err := process.Name()
if err != nil {
    // Ignore the error from process.Name() because errors
    // come from processes which are not owned by the current user
    // on Mac - https://github.com/shirou/gopsutil/issues/1288
    continue
}